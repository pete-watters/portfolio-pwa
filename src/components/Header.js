import React from 'react';
import { NavLink } from 'react-router-dom';
import Nav from './Nav';
import CONFIG from '../constants';
// TODO fix this to highlight when on /
const Header = () =>
  <header>
    <h1><NavLink href={CONFIG.ROUTES.HOME}>Peter Watters</NavLink></h1>
    <Nav />
  </header>;

export default Header;