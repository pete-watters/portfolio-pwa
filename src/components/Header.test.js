import React from 'react';
import { shallow } from 'enzyme';
import Header from './Header';

describe('Header', () => {
  it('should render the component correctly', () => {
    expect(shallow(<Header />)).toMatchSnapshot();
  });
});
