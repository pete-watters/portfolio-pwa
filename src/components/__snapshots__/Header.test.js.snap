// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render the component correctly 1`] = `
<header>
  <h1>
    <NavLink
      to="/"
    >
      Pete Watters
    </NavLink>
  </h1>
  <Nav />
</header>
`;
